civis/response.py

```diff
@@ -22,19 +22,23 @@ def _response_to_json(response):
 
     Returns
     -------
-    dict
-        The data in the response body.
+    dict | None
+        The data in the response body or None if the response has no
+        content.
 
     Raises
     ------
     CivisClientError
         If the data in the raw response cannot be parsed.
     """
-    try:
-        return response.json()
-    except ValueError:
-        raise CivisClientError("Unable to parse JSON from response",
-                               response)
+    if response.status_code in [204, 205]:
+        return None
```

Comment:
Returning an empty dict here would mean that a list endpoint with no results is not distinguishable from an endpoint that returns no content.  I think we should keep these distinguishable, and wrap the original response as thinly as possible, but I'm open to other opinions.